### PR TITLE
Gradle deprecation fix1.0

### PR DESCRIPTION
**Pull request by arunvenmany-ibm**
For resolving https://github.com/OpenLiberty/ci.gradle/issues/900

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.5-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.10-bin.zip
```
Comment (cherylking) [inline gradle/wrapper/gradle-wrapper.properties:3]: Let's change this back to 8.5 for the build to ensure we do not require an upgrade with these changes.

File: src/test/groovy/io/openliberty/tools/gradle/InstallLiberty_DefaultNoMavenRepo.groovy [hunk 2]
```diff
@@ -45,6 +45,7 @@ class InstallLiberty_DefaultNoMavenRepo extends AbstractIntegrationTest{
             .buildAndFail()
 
         String output = result.getOutput()
-        assert output.contains("org.gradle.api.artifacts.ResolveException") : "Expected installLiberty to fail with ResolveException"
+        //gradle 8.10 is throwing TypedResolveException instead of ResolveException
+        assert output.contains("org.gradle.api.internal.artifacts.ivyservice.TypedResolveException") : "Expected installLiberty to fail with TypedResolveException"
     }
```
Comment (cherylking) [inline src/test/groovy/io/openliberty/tools/gradle/InstallLiberty_DefaultNoMavenRepo.groovy:50]: Change this to the other exception that both 8.5 and 8.10 had in common in the stack trace (ModuleVersionException?).

File: src/test/resources/sampleJSP.servlet/testCompileJSP17.gradle [hunk 3]
```diff
@@ -44,6 +44,12 @@ liberty {
         deploy {
             apps = [war]
         }
+        undeploy {
+            apps = [war]
+        }
+        features {
```
Comment (cherylking) [inline src/test/resources/sampleJSP.servlet/testCompileJSP17.gradle:50]: remove this change

**Review by cherylking — CHANGES_REQUESTED** [review]

File: src/test/resources/sampleJSP.servlet/testCompileJSP17.gradle (hunk 3 above)
Comment (arunvenmany-ibm) [reply src/test/resources/sampleJSP.servlet/testCompileJSP17.gradle]: Done

File: src/test/groovy/io/openliberty/tools/gradle/InstallLiberty_DefaultNoMavenRepo.groovy (hunk 2 above)
Comment (arunvenmany-ibm) [reply src/test/groovy/io/openliberty/tools/gradle/InstallLiberty_DefaultNoMavenRepo.groovy:47]: Changed

File: gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (arunvenmany-ibm) [reply gradle/wrapper/gradle-wrapper.properties]: Done

**Review by cherylking — APPROVED** [review]

**Pull request merged**